sql-parser/dialect/mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/MySQLStatementVisitor.java

```diff
@@ -1611,6 +1611,11 @@ private ASTNode createProjection(final ProjectionContext ctx, final AliasSegment
             result.setAlias(alias);
             return result;
         }
+        if (projection instanceof BetweenExpression) {
```

Comment:
Hi @iamhucong, can you add sql parser test case?

Reply:
Okay, I will continue working on this.